spec/models/user_spec.rb

```diff
@@ -22,6 +22,15 @@
 
     it { is_expected.to validate_presence_of(:session_token) }
     it { is_expected.to validate_presence_of(:session_expiry) }
+    it { is_expected.to validate_presence_of(:language) }
+
+    it { is_expected.to validate_length_of(:name).is_at_least(2) }
+    it { is_expected.to validate_length_of(:name).is_at_most(255) }
+
+    it { is_expected.to validate_length_of(:email).is_at_least(5) }
+    it { is_expected.to validate_length_of(:email).is_at_most(255) }
```

Comment:
Don't forget to make the changes here as well. you can do it in one line

Reply:
Okay on it